all_test.go

```diff
@@ -500,4 +534,80 @@ func TestSessionDelete(t *testing.T) {
 	if err != ErrInvalidSessionToken {
 		t.Error("Expected ErrInvalidSessionToken after Logout")
 	}
+
+	teardown(c)
+}
+
+func TestUpdateIdentity(t *testing.T) {
+	c := setup1(t)
+
+	newEmail := "newEmail"
+	newUsername := "newUsername"
+	newName := "newName"
+	newSurname := "newSurname"
+	newMobile := "newMobile"
+
+	if err := c.UpdateIdentity(notFoundUserId, newEmail, newUsername, newName, newSurname, newMobile); err != nil {
```

Comment:
The logic is hard to follow here. It would be clearer if it read:
```go
if err := c.UpdateIdentity(...); err != ErrIdentityAuthNotFound {
    t.Fatalf("Expected ErrIdentityAuthNotFound, but instead got %v", err)
}
```